dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js

```diff
@@ -0,0 +1,36 @@
+var messages = {
+    WELCOME_OPEN_LIBERTY: "Welcome to Open Liberty",
+    OPEN_LIBERTY_APP_SERVER: "Open Liberty Application Server",
+    WEBSPHERE_OPEN_LIBERTY_LOGO: "WebSphere Liberty Logo",
+    WELCOME_LIBERTY_SECTION: "Welcome to ",
+    WELCOME_IBM_OPEN_SOURCE: "An IBM Open Source Project",
+    RESOURCES: "Resources",
+    GROUPS_IO_FORUM: "Groups.io forum",
+    COPYRIGHT_OPEN_LIBERTY: "Licensed Materials — Property of the Open Liberty project, Copyright IBM Corp and others 1997, 2019. Project is distributed under the terms of the Eclipse Public License v1.",
+    WELCOME: "Welcome to Liberty",
+    WEBSPHERE_LIBERTY_LOGO: "WebSphere Liberty Logo",
+    WEBSPHERE_OPEN_LIBERTY_FOOTER: "WebSphere Liberty footer Logo",
+    WELCOME_SECTION: "Welcome Section",
+    WELCOME_SECTION_CONTENT: "Welcome Section Content",
+    WEBSPHERE_APPSERVER: "WebSphere Application Server Beta",
```

Comment:
This doesn't seem applicable to Open Liberty either? Where/when is it used?

Reply:
Suggest that this file contain only those messages which are actually used by OL.

Reply:
I updated the messages.js to contain only messages from OL. Thanks!